.travis.yml

```diff
@@ -21,7 +21,7 @@ cache:
     - $HOME/.composer/cache
 
 before_install:
-  - ./travis/before_install.sh
+  - composer self-update
```

Comment:
@padraic Fine with me, warnings are annoying and ~ 3 s are ok. No need to flame though, it's opinion based and OT to this PR.

Any issues related to PR topic?